.github/workflows/iroha2_base.yml

```diff
@@ -1,20 +1,17 @@
-name: I2::Base Image Deploy
+name: I2Base::Publish
```

Comment:
```suggestion
name: I2::CI::Base::Publish
```

Reply:
`I2::Base::Publish` because the images provide base environments to build. CI is just one of the uses